load/UpdateDBPLanguageTranslation.py

```diff
@@ -23,26 +24,36 @@ def updateOrInsertlanguageTranslation(self):
         return self.process()
 
     def process(self):
-        bibles = self.languageReader.getBibleIdMap()
+        languageRecords = self.languageReader.resultSet
 
-        for bibleId in bibles:
-            for _, languageRecord in bibles.get(bibleId):
-                lang = self.languageId(bibleId, languageRecord)
+        for languageRecord in languageRecords:
+            lang = self.languageId(languageRecord)
 
-                if lang != None:
-                    if languageRecord.HeartName() != None:
-                        self._processLanguageWithHeartName(lang, languageRecord)
+            if lang != None:
+                self._populatePriorityNine(lang, languageRecord)
 
-                    if languageRecord.AltName() != None:
-                        self._processLanguageWithAltName(lang, languageRecord)
+                if languageRecord.AltName() != None:
+                    self._processLanguageWithAltName(lang, languageRecord)
 
         return True
 
-    def _processLanguageWithHeartName(self, languageSourceId, languageRecord):
-        heartNamePriority = self.HEART_NAME_PRIORITY
-        languageTranslationId = languageSourceId
-        languageNames = [languageRecord.HeartName()]
-        self._updateOrInsertLanguageTranslations(languageNames, languageSourceId, languageTranslationId, heartNamePriority)
+    def _populatePriorityNine(self, languageSourceId, languageRecord):
+        if languageRecord.LangName() != None:
+            priorityEngLanguageName = [languageRecord.LangName().strip()]
+            self._updateOrInsertLanguageTranslations(
```

Comment:
for priority 9, the indexed key is different than for lower priorities. This may mean we need to pass in the key to updateOrInsertLanguageTranslations
if 9, the index is languageSourceId, languageTranslationId, Priority. So if there is an existing name for this tuple, we want to update it. There should be only one name value for priority 9 for a source_id, translation_id pair

Reply:
@bradflood , ok, I understand. Let me change it.

Reply:
@bradflood , I have changed it. I have create a new method to handle the language record with priority equals to 9. The new method is named: `_updateOrInsertLangTranslationsHighPriority` you can see it in the following line:
https://github.com/faithcomesbyhearing/dbp-etl/pull/92/files#diff-d21d9d4001c386f3021901b139bfa9036284349ef9f6a39ccb598a08c7234787R103